libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -303,6 +306,14 @@ public void hideRecenterBtn() {
     recenterBtn.hide();
   }
 
+  @Override public void showSpeed() {
+    speedLimitView.show();
+  }
+
+  @Override public void hideSpeed() {
```

Comment:
Should we call this `hideSpeedLimit` or even `hideSpeedLimitView`?

Reply:
Removed this method